src/components/InfiniteScroll/index.js

```diff
@@ -23,15 +23,13 @@ const messages = defineMessages({
 @observer
 export default class InfiniteScroll extends Component {
   static propTypes = {
-    hasMore: PropTypes.bool,
```

Comment:
dont you need this to know if you need to call `loadMore()`? we shouldn't call `loadMore()` if the we have items == totalCount

Reply:
just want to make sure when the user scrolls to the end we aren't repeatedly making an unnecessary network call since we know we have all of them.

Reply:
in loadmore it checks hasMore, and we have seperate hasmore for different type of query, so better remove.

Reply:
got it